src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DetectPreviewFeatureUnitTests.cs

```diff
@@ -116,8 +114,7 @@ public async Task TestArrayOrPreviewTypes()
             var csInput = @" 
         using System.Runtime.Versioning; using System;
         namespace Preview_Feature_Scratch
-        {" +
-@"
+        {
             public class Program
             {
                 static void Main(string[] args)
```

Comment:
Actually I tried that already. I expected it to work like `Lib[] array1 = {|#1:new() { }|};` since we're listening for an `OperationKind.ObjectCreation`, but it led to a compile error: `error CS8752: The type 'Lib[]' may not be used as the target type of new()`

Reply:
Ah, interesting. Do you have a test case elsewhere that allows `new()` to be used?

Reply:
Added 1 now to this test case.